### ISSUE TITLE
Solar_System and Game_Of_Life not working correctly

### ISSUE THREAD

**Issue by Nickiel12** [iced-rs/iced#1249]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this issue related to iced?

- [X] My hardware is compatible and my graphics drivers are up-to-date.

### What happened?

When running the examples 
`cargo run --package game_of_life` and `cargo run --package solar_system` 
the created windows don't display correctly. This is an exact replication of the symptoms of #342 down to the fact that when I turned anti-aliasing off, they worked fine. 
I am running Arch linux, `iced="0.3"`, and rustc version is 1.57.0

### What is the expected behavior?

For solar_system to display anything other than a white screen, and game_of_life to show the game grid.

### Version

0.3.0

### Operative System

Linux

**Comment by hecrj** [comment]
- Do the `wgpu` examples work for you?
- Could you try to see if `master` has the same issue?
- Could you share your hardware specifications?

**Comment by Nickiel12** [comment]
Yes, I did not have any problems running the wgpu examples (several of them anyway, I didn't run them all). 
I believe I installed `master` by accident now I think about it, I just cloned the repo.

CPU: AMD Ryzen 5 2500U with Radeon Vega Mobile Gfx (8) @ 2.000GHz 
GPU: AMD ATI Radeon Vega Series / Radeon Vega Mobile Series

**Comment by hecrj** [comment]
It seems like you have a dual GPU setup.

It's very likely the `wgpu` examples are running on your iGPU and not the dedicated one. When you enable the `antialiasing` flag, `iced` will try to choose a high performace card.

To test this, could you try running the `wgpu` examples again but setting the `WGPU_POWER_PREF` environment variable to the value `high`?

**Comment by Nickiel12** [comment]
I ran ` export WGPU_POWER_PREF=high` before `cargo run bunnymark` it error-ed out with `value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', wgpu-info/src/main.rs:150:18`

**Comment by hecrj** [comment]
It seems your high performance card is using the OpenGL backend of `wgpu`.

This backend is still considered a best-effort support. And so we've confirmed this is a hardware compatibility issue.

If you want to use the `antialiasing` flag, I recommend you to use the `glow` renderer.
